### ISSUE TITLE
Accessibility improvements to exercism.io

### ISSUE THREAD

**Issue by Tetsuro** [exercism/exercism.io#3066]
Hey guys! 👋 I would like to suggest some quick a11y* improvements to the exercism.io website.

## 1. Add more landmarks

<img width="1252" alt="exc1" src="https://cloud.githubusercontent.com/assets/351842/17826132/c83fac6c-663b-11e6-97a7-a897186b059a.png">

Currently only `navigation` can be found under the landmarks menu. We can add more for the main area, the sidebar, and the footer to make it easier to navigate in screen readers. 

## 2. Remove some screen reader noise

I think we can hide a lot of things from screen readers, such as the horizontal rule and any decorative images.

## 3. Give alt text to images, or blank alt text

![image](https://cloud.githubusercontent.com/assets/351842/17826235/a2389e24-663c-11e6-8b8e-14041f9e2324.png)

Since there is not alt text on a lot of the images, I might suggest giving them some alt text. Alternatively, since the fact that it's a logo is not really important (it's more of a decorative image), we can give it a blank `alt` text so that the screen reader does not read them at all. 

---

@bernardoamc I can totally help out with this, if you'd like! Let me know :) Thanks!

a11y: shortened form of "accessibility"

**Comment by bernardoamc** [comment]
This is awesome @Tetsuro !! I would love to see these changes happening, do you want to tackle this in a PR? Just let me know if you need any help! Thanks! 🎉

**Comment by Tetsuro** [comment]
@bernardoamc sure thing - I'd love to tackle this! I'll make a PR for it :)

**Comment by kytrinyx** [comment]
Holy smokes this is fantastic. Thank you :heart: :heart:

FYI @troubalex

**Comment by gusseting** [comment]
@Tetsuro I'm looking at making changes to the homepage in the not too distant future with the following discussion, and it would be great to have you take part re: accessibility.

**Comment by Tetsuro** [comment]
@gusseting Sounds great, count me in! Definitely not an expert but would love to be part of the discussion.

**Comment by edward** [comment]
@Tetsuro you’re a champ for taking this on. Really happy to see you pop up here.

**Comment by bernardoamc** [comment]
We can close this issue now, thank you @Tetsuro ! ❤️
